velox/duckdb/conversion/DuckParser.cpp

```diff
@@ -99,6 +99,21 @@ std::string duckOperatorToVelox(ExpressionType type) {
   }
 }
 
+// SQL functions could be registered with different prefixes.
+// This function returns a full Velox function name with registered prefix.
+std::string toFullFunctionName(
+    const std::string& functionName,
+    const std::string& prefix) {
+  // Some functions are not registered with a prefix.
```

Comment:
These are called special forms. Let's use that term in the comments and add static constant set.

```
static const std::unordered_set<std::string> specialForms{"cast", "and",..};

if (specialForms.contains(functionName)) {
...
```

Reply:
Fixed.